integration/jpa-base/pom.xml

```diff
@@ -114,6 +125,7 @@
                                         requires java.sql;
                                         requires java.persistence;
                                         requires com.blazebit.persistence.core;
+                                        exports com.blazebit.persistence.integration.jpa;
```

Comment:
Does this make the entry below superfluous?

Reply:
No